src/material.cpp

```diff
@@ -843,7 +843,8 @@ void pyne::Material::from_text(std::string filename) {
     } else if (keystr == "APerM") {
       f >> valstr;
       atoms_per_molecule = pyne::to_dbl(valstr);
-    } else if (pyne::nucname::isnuclide(keystr)) {
+    } else if (pyne::nucname::isnuclide(keystr) ||
+               pyne::nucname::iselement(keystr)) {
```

Comment:
It seems like is the only line actually changed (other than whitespace)? I'm not sure what this is trying to solve, but it looks fine to me. If we actually want this bool to be true for either isnuclide or iselement, then I think we should pull it. If you can confirm this @scopatz , I'll pull it for you.